src/tools/ip/quicip.h

```diff
@@ -17,15 +17,28 @@
 #pragma once
 
 #include <msquic.h>
+#include <msquicp.h>
```

Comment:
What is this using out of msquicp? (Other then the self signed security thing).

Reply:
That's it.